scripts/cybershake/install_cybershake_fault.py

```diff
@@ -64,6 +72,31 @@ def main():
     )
 
 
+def gen_args_cmd(
+    process: ProcessType,
+    sim_dir,
+    command_template,
+    template_parameters,
+    add_args={},
+):
+    #this function is adapted from generate_command from shared_workflow.shared_template
+    command_parts = []
+
+    command_parts = command_template.format(**template_parameters).split()
+    #remove srun, python, and *.py from the command
+    for i in list(command_parts):
+        if (any([ i == x for x in ['srun','python']]) or i[-3:] == '.py'):
+            command_parts.remove(i)
+            
+    for key in add_args:
+        command_parts.append("--" + key)
+        if add_args[key] is True:
+            continue
+        command_parts.append(str(add_args[key]))
+
+    return (list(map(str, command_parts)))
```

Comment:
All elements of command_parts seem to be a string already?

Reply:
I think its better to keep it there, as it is not computational heavy (can only be so much args) and work as a last safety check if any unexpected values are in there (very unlikely but better be safe than sorry).